civihr_employee_portal/js/reports.js

```diff
@@ -797,9 +797,6 @@
         this.placeholderFormat = 'dd/MM/yyyy';
         this.date = new Date();
         this.filtersCollapsed = true;
-      })
```

Comment:
Could you please explain why this controller is not necessary anymore?

Reply:
> Could you please explain why this controller is not necessary anymore?

`SettingsController` ? (sorry, can't see which one). There's an explanation in the description for that one.

Reply:
Please skip that comment.